.gitignore

```
@@ -0,0 +1,16 @@
+# Ignore all files
+*
+
+# License and environment files
+!LICENSE
+!Makefile
+!Pipfile
+!Pipfile.lock
+!README.md
+!.gitignore
+
+# Program files
+!gpgopt.py
+!scmail.py
+!app.py
+!cli.py
```

Nit: Add a newline to the end of this file.
```suggestion
!cli.py

```

---

Thanks, I added one line.